### DISCUSSION TITLE
Optionally keep off-canvas content, minor inconsistencies.

### DISCUSSION THREAD

**Discussion by ShaKabosh** [Orama-Interactive/Pixelorama#779] [Ideas]
Hi! First off, I'd like to say that I love this project a ton! One thing that bugs me though is the deletion of off-canvas content when using the move tool (when you move a drawing partially off the canvas, it deletes the part that is no longer on the canvas). I have also noticed a few minor inconsistencies, such as the labelling of undoing shading as "Undo Draw" as opposed to "Undo Shading". These are just quality of life nit picks: the project is still brilliant! Thanks devs and contributors for making this program!

**Comment by novhack** [comment]
I understand what you mean but I don't know about any bitmap drawing program which would actually keep canvas content outside of canvas. If that's bothering you I recommend to make the canvas a bit bigger than the final image you want to export. Like that you can create a rectangle inside of the bigger canvas to draw in which you can crop out on export.
